CategorySuggest.body.php

```diff
@@ -95,17 +90,13 @@ function fnCategorySuggestSaveHook( $m_isUpload, $m_pageObj ) {
 	 	foreach( $arrSelectedCats as $m_cat ) {
 	 	 	if(strlen($m_cat)>0){
 				$m_cat = Title::capitalize($m_cat, NS_CATEGORY);
-				$m_text .= "\n[[$m_catString:" . trim($m_cat) . "]]";
+				$m_text .= "\n[[". $m_catString .":" . mysql_escape_string(trim($m_cat)) . "]]";
```

Comment:
@AndySky21 , pull #18 looks like some nice cleanup. I haven't had a chance to test yet, but a cursory review only turned up one problem: the addition of `mysql_escape_string()` I'm not sure if this was coming in from a merge-resolution with master and can just be removed, but if not we'll need to find a different way to accomplish the goal as not everyone uses MySQL (we had bug reports mentioning running MediaWiki on SQLITE and Postgres as well).